data/json/recipes/food/frozen.json

```diff
@@ -128,6 +128,25 @@
       [ [ "water_clean", 1 ] ]
     ]
   },
+  {
+    "result": "frozen_lemonade",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": "10 m",
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 3 } ],
+    "tools": [ [ [ "rock_quern", -1 ], [ "clay_quern", -1 ], [ "mortar_pestle", -1 ], [ "food_processor", 10 ] ] ],
+    "components": [
+      [ [ "milk_standard", 1, "LIST" ], [ "con_milk", 1 ], [ "can_coconut", 1 ] ],
```

Comment:
looks like you left the milk in the recipe, I'd remove the milk and add a requirement for one real lemon or irradiated lemon per portion in addition to the lemonade

Reply:
Oops! I was thinking of the chik-fil-a frozen lemonade (which has dairy) and never bothered to look up the real thing! Fixed.